src/main/java/com/danit/models/Contract.java

```diff
@@ -0,0 +1,140 @@
+package com.danit.models;
+
+
+import com.danit.utils.CustomDateAndTimeDeserialize;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import java.util.Date;
+
+@Entity
+@Table(name = "contracts")
+public class Contract {
+  @Id
+  @Column(name = "id")
+  private Long id;
+
+  @Column(name = "start_date")
+  @JsonDeserialize(using = CustomDateAndTimeDeserialize.class)
+  @JsonFormat(
+      shape = JsonFormat.Shape.STRING,
+      pattern = "dd-MM-yyyy")
+  @Temporal(TemporalType.DATE)
+  private Date startDate;
+
+  @Column(name = "end_date")
+  @JsonDeserialize(using = CustomDateAndTimeDeserialize.class)
+  @JsonFormat(
+      shape = JsonFormat.Shape.STRING,
+      pattern = "dd-MM-yyyy")
+  @Temporal(TemporalType.DATE)
+  private Date endDate;
+
+  @Column(name = "credit")
+  private Float credit;
+
+  @Column(name = "active")
+  private boolean isActive;
+
+  @ManyToOne(fetch = FetchType.EAGER, cascade = CascadeType.MERGE)
+  @JoinColumn(name = "client_id", updatable = false, insertable = false)
+  @JsonIgnore
+  private Client client;
+
+  @Column(name = "package_id")
+  private Long packageId;
+
+  @Column(name = "client_id")
+  private Long clientId;
+
+  public Long getId() {
+    return id;
+  }
+
+  public void setId(Long id) {
+    this.id = id;
+  }
+
+  public Contract() {
+  }
+
+  public Date getStartDate() {
+    return startDate;
+  }
+
+  public void setStartDate(Date startDate) {
+    this.startDate = startDate;
+  }
+
+  public Date getEndDate() {
+    return endDate;
+  }
+
+  public void setEndDate(Date endDate) {
+    this.endDate = endDate;
+  }
+
+  public Float getCredit() {
+    return credit;
+  }
+
+  public void setCredit(Float credit) {
+    this.credit = credit;
+  }
+
+  public boolean getIsActive() {
+    return this.isActive;
+  }
+
+  public void setIsActive(boolean isActive) {
+    this.isActive = isActive;
+  }
+
+  public Client getClient() {
+    return client;
+  }
+
+  public void setClient(Client client) {
+    this.client = client;
+  }
+
+  public Long getPackageId() {
+    return packageId;
+  }
+
+  public void setPackageId(Long packageId) {
+    this.packageId = packageId;
+  }
+
+  public Long getClientId() {
+    return clientId;
+  }
+
+  public void setClientId(Long clientId) {
+    this.clientId = clientId;
+  }
+
+  @Override
+  public String toString() {
```

Comment:
Perhaps you need to re-generate toString() method

Reply:
no)